components/charts/hsAverage.js

```diff
@@ -0,0 +1,63 @@
+import { db } from "../../firebaseConfig";
+import { Line } from "react-chartjs-2";
+
+const backup = db.collection("1A Data").doc("HS Average");
+const dataVal = [];
+const dataLabel = [];
+const dataColor = [];
+
+const observer = backup.onSnapshot(
```

Comment:
discussion: should logic for components be added in a separate file and passed in here when necessary?

Reply:
yeah, but this is def to be discussed later, we haven't determined atm the specifics